src/modules/auth/controllers/auth.controller.ts

```
@@ -0,0 +1,42 @@
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  // Registration endpoint
+  @Post('register')
+  @UsePipes(new ValidationPipe({ whitelist: true }))
+  async register(@Body() registerDto: RegisterDto) {
+    const user = await this.authService.register(
+      registerDto.username,
+      registerDto.password,
+    );
+    const { password, ...result } = user;
```

ESLint

[error] 'password' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)